src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/LogRationalApproximationIndividual.cpp

```diff
@@ -7,36 +7,34 @@
 #include <shogun/lib/config.h>
 
 #include <shogun/base/Parameter.h>
-#include <shogun/lib/SGVector.h>
 #include <shogun/lib/SGMatrix.h>
-#include <shogun/mathematics/linalg/linsolver/LinearSolver.h>
+#include <shogun/lib/SGVector.h>
+#include <shogun/mathematics/eigen3.h>
```

Comment:
I think the eigen3 includes can go once you use linalg

Reply:
solved